app/components/subcases/subcase-header.js

```diff
@@ -24,17 +24,17 @@ export default Component.extend({
   caseToDelete: null,
   onMoveSubcase: null, // from args, the parent route subcases needs to be refreshed after subcase is moved/deleted
 
-  canPropose: computed('subcase.{requestedForMeeting,hasActivity}', 'isAssigningToOtherAgenda', async function() {
+  canPropose: computed('subcase.{requestedForMeeting,agendaActivities}', 'isAssigningToOtherAgenda', async function() {
```

Comment:
Is it an option to refactor this component to a Glimmer component in this PR or will that lead us too far?

Reply:
well nevermind, it's done now :D